Dfe.Data.SearchPrototype/Web/Views/Home/_pagination.cshtml

```diff
@@ -0,0 +1,101 @@
+﻿@* @* partial view for pagination*@
+<style>
+    button.paginationButton {
+        background: none;
+        border: none;
+        color: blue;
+        text-decoration: underline;
+        cursor: pointer;
+        font-size: 17px;
+        padding-block: 0px;
+        padding-inline: 0px;
+    }
+</style>
+
+@if (Model.IsPageable)
+{
+    <nav id="pagination" class="govuk-pagination" role="navigation" aria-label="results-pagination">
+        <ul class="govuk-pagination__list">
+            <li class="govuk-pagination__item
+                @(Model.IsFirstPage ? "govuk-visually-hidden" : "")">
+                <button type="submit"
+                        name="PageNumber"
+                        value=@Model.PreviousPageNumber
```

Comment:
i've looked into this and using asp-for for buttons is not recommended, we could do a workaround and use hidden input with asp-for, however because we have one form with different submissions - using name value in a button allows us to send different values  for same action (ie.PageNumber)